dist-persist/wbstack/src/Internal/WbStackPlatformReservedUser.php

```diff
@@ -131,10 +130,31 @@ public static function getOAuthConsumer($consumerName, $version) {
             return false;
         }
 
-        return [
+        $data = [
             'agent' => $c->getName(),
             'consumerKey' => $c->getConsumerKey(),
             'consumerSecret' => \MediaWiki\Extension\OAuth\Backend\Utils::hmacDBSecret( $c->getSecretKey() ),
         ];
+
+        $a = \MediaWiki\Extension\OAuth\Backend\ConsumerAcceptance::newFromUserConsumerWiki(
+            $db,
+            $user->getId(),
+            $c,
+            $c->getWiki(),
+            \MediaWiki\Extension\OAuth\Backend\ConsumerAcceptance::READ_NORMAL,
+            $c->getOAuthVersion(),
+        );
+
+        if ( $a !== false ) {
```

Comment:
I started investigating this locally but I can also ask the question: is this also going to return valid accessKey's and accessSecrets for non-owner only tokens? If so this is probably undesirable. Not because it will break non-owner only consumers but it could be undesirable for credentials for the platform reserved user to be given to these tools (e.g. confusing for future users / mild security risk)

Reply:
Perfect